src/vorta/utils.py

```diff
@@ -202,11 +202,16 @@ def get_private_keys() -> List[str]:
                 continue
             try:
                 if is_ssh_private_key_file(key_file):
-                    available_private_keys.append(key)
+                    if os.stat(key_file).st_mode & 0o077 == 0:
+                        available_private_keys.append(key)
+                    else:
+                        logger.warning(f'Permissions for {key_file} are too open.')
+                else:
+                    logger.debug(f'Not a private SSH key file: {key}')
             except PermissionError:
                 # Handling PermissionError separately from OSError because it can be safely ignored
                 # (if the user doesn't have permission to read the file, it's not an unexpected error).
```

Comment:
This comment line is no longer up-to-date.

Reply:
Do I need the comment now? (The warning seems self-explanatory, we handled PermissionError separately to let the user know about an expected error)

Reply:
You are right it is not needed any more.